pandoc-core/pandoc-core.cabal

```diff
@@ -0,0 +1,127 @@
+                     , network               >= 2.6
+                     , network-uri           >= 2.6      && < 2.8
+                     , pandoc-types          >= 1.22.2   && < 1.23
+                     , parsec                >= 3.1      && < 3.2
+                     , random                >= 1        && < 1.3
+                     , safe                  >= 0.3.18   && < 0.4
+                     , skylighting           >= 0.13     && < 0.14
+                     , tagsoup               >= 0.14.6   && < 0.15
+                     , text                  >= 1.1.1.0  && < 2.1
+                     , time                  >= 1.5      && < 1.14
+                     , yaml                  >= 0.11     && < 0.12
+library
+  import:              common-options
+                     , common-library
+  build-depends:       Glob                  >= 0.7      && < 0.11
+                     , aeson-pretty          >= 0.8.9    && < 0.9
+                     , base64                >= 0.4      && < 0.5
+                     , case-insensitive      >= 1.2      && < 1.3
+                     , connection            >= 0.3.1
+                     , gridtables            >= 0.1      && < 0.2
+                     , http-client           >= 0.4.30   && < 0.8
+                     , http-client-tls       >= 0.2.4    && < 0.4
+                     , mime-types            >= 0.1.1    && < 0.2
+                     , unicode-collation     >= 0.1.1    && < 0.2
+                     , unicode-transforms    >= 0.3      && < 0.5
+                     , unicode-collation
+                     , unicode-transforms
+                     , zip-archive           >= 0.2.3.4  && < 0.5
+  exposed-modules:     Text.Pandoc.Asciify
+                     , Text.Pandoc.Class
+                     , Text.Pandoc.Class.CommonState
+                     , Text.Pandoc.Class.IO
+                     , Text.Pandoc.Class.PandocIO
+                     , Text.Pandoc.Class.PandocMonad
+                     , Text.Pandoc.Class.PandocPure
+                     , Text.Pandoc.Class.Sandbox
+                     , Text.Pandoc.Error
+                     , Text.Pandoc.Extensions
+                     , Text.Pandoc.Highlighting
+                     , Text.Pandoc.Logging
+                     , Text.Pandoc.MIME
+                     , Text.Pandoc.MediaBag
+                     , Text.Pandoc.Options
+                     , Text.Pandoc.Parsing
+                     , Text.Pandoc.Parsing.Capabilities
+                     , Text.Pandoc.Parsing.Citations
+                     , Text.Pandoc.Parsing.General
+                     , Text.Pandoc.Parsing.GridTable
+                     , Text.Pandoc.Parsing.Lists
+                     , Text.Pandoc.Parsing.Math
+                     , Text.Pandoc.Parsing.Smart
+                     , Text.Pandoc.Parsing.State
+                     , Text.Pandoc.Parsing.Types
+                     , Text.Pandoc.Readers.LaTeX.Types
```

Comment:
Why this?  It seems LaTeX-specific and thus not a candidate for core, but I assume something else here depends on it?

Reply:
`Macro` from that module is uesd by Text.Pandoc.Parsing.{State,Capabilities}. There's probably a better solution, this is all very experimental.